libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.ts

```diff
@@ -213,4 +213,13 @@ export class SkyVerticalTabsetComponent
   protected tabGroupsArrowUp(): void {
     this.adapterService.focusPreviousButton(this.tabGroups);
   }
+
+  protected showTabs(): void {
+    this.tabService.showTabs();
+
+    // Wait for tabs to finish opening.
+    setTimeout(() => {
```

Comment:
Could you use the `@tabGroupEnter.done` callback to better ensure that the content is done instead of the generic `setTimeout`?

Reply:
good call! this simplified the solution a Lot